homework/input-output/src/main/java/com/cbfacademy/App.java

```diff
@@ -1,7 +1,46 @@
 package com.cbfacademy;
 
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
 public class App {
-    public static void main(String[] args) {
-        System.out.println("Hello World!");
-    }
+    public static void main(String[] args) {  
+    
+    // Create program that loads file and prints each value to the file.
+    Properties example = new Properties();
+    try {
+        FileInputStream input = new FileInputStream("C:\\Users\\Admin\\cbfacademy\\java-exercises-YOkolo24\\homework\\input-output\\src\\main\\resources\\example.properties");
+            example.load(input);
+            example.forEach((key, value) -> System.out.println(key + " = " + value));
+
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
```

Comment:
This behaviour should live inside its own method, which you then call from `main`